var/spack/repos/builtin/packages/opium/package.py

```diff
@@ -16,6 +16,7 @@ class Opium(Package):
 
     depends_on('blas')
     depends_on('lapack')
+    build_targets = ['clean']
```

Comment:
Normally the build phase runs `make`, which is equivalent to `make all`. Now you're running `make clean`. Is that what you want?

Reply:
Yes,
Since make all is executed, the behavior is different from before the correction and an error has occurred.
Added build_targets = ['clean'] as a workaround.

Reply:
My fix wasn't appropriate.
I will rethink how to fix it.